README.md

```diff
@@ -28,11 +36,13 @@ Checkout the project and type `./gradlew build`. You find the executable JAR in
      java version "1.8.0_144"
      Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
      Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed model
-     $ git clone https://github.com/AlloyTools/org.alloytools.alloy.git
-     $ cd org.alloytools.alloy
+     $ git clone --recursive https://github.com/haslab/Electrum2.git
+     $ cd Electrum2
      $ ./gradlew build
      $ java -jar org.alloytools.alloy.dist/target/org.alloytools.alloy.dist.jar
      # opens GUI
+     
+The `master` branch points the latest stable version, 2.0. For stable 2.1 release candidates, branch `v2.1` should be cloned instead. 
```

Comment:
Is this paragraph up to date?  The current version of Alloy is 5.0, so 2.0 and 2.1 doesn't sound right.